src/components/pages/userDetailPage.tsx

```diff
@@ -1,7 +1,44 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+
+import { User } from 'devu-shared-modules'
 
 import PageWrapper from 'components/shared/layouts/pageWrapper'
+import LoadingOverlay from 'components/shared/loaders/loadingOverlay'
+
+import { useParams } from 'react-router-dom'
+
+import RequestService from 'services/request.service'
+
+import EditUserForm from 'components/shared/forms/editUserForm'
+
+import styles from './userDetailPage.scss'
+
+type Params = {
+  userId: string
+}
+
+const UserDetailPage = ({}) => {
+  const { userId } = useParams() as Params
+
+  const [loading, setLoading] = useState(true)
+  const [user, setUser] = useState({} as User)
+
+  useEffect(() => {
+    RequestService.get(`/api/users/${userId}`)
+      .then(setUser)
+      .catch((e) => console.error(e))
```

Comment:
this should catch the error, then set it. Then return `<ErrorPage error={error} />` if error is not null. Take a look at the `UserCoursesListPage` to see what that looks like.

Reply:
yes this does mean that this is now setup where the error page display is not recoverable